lessons/linear-regression/langs/en-us/index.adoc

```diff
@@ -15,165 +15,141 @@
-- Reflect on the scatterplots you have created. Can we _predict_ an animal's adoption time based on its size? Its age?
+Have students turn to Part 1 of @printable-exercise{lr-plot-intro.adoc} and consider the scatter plots for weeks-v-pounds and weeks-v-age.
 
-Have students write down what they think, then quickly survey the class.
+[cols="1a,1a", grid="none", frame="none"]
+|===
+| @image{images/pounds-v-weeks.gif,"weeks-v-pounds scatter plot with a tight cluster in the lower left and some points toward the right and top right corner", 400}
+| @image{images/age-v-weeks.gif,"weeks-v-age scatter plot with some points in the lower left and some in the upper right", 400}
+|===
 
-@right{@image{images/pounds-v-weeks.gif,"weeks-v-pounds scatter plot with a cluster of points on the lower left and some points in the upper right", 300}}
-We are asking if we can use an animal’s size or age to predict how long it will take to be adopted. A scatter plot of adoption time versus size _does_ suggest that smaller animals get adopted faster than larger animals. Similarly, younger animals tend to be adopted faster than older ones.
+[.lesson-instruction]
+--
+Quickly survey the class about whether they think we can use an animal’s size or age to predict how long it will take to be adopted.
+
+- _A scatter plot of adoption time versus size does suggest that smaller animals get adopted faster than larger animals._
+- _Similarly, younger animals tend to be adopted faster than older ones._
+--
 
 Can we be more precise about this, and actually _predict_ how long it will take an animal to be adopted, based on these factors? And which one would give us a better prediction?
 
-The mean, median, and mode are three different ways to measure the “center” of a dataset in one dimension. Each represents a different way to collapse a bunch of points on a number line into a single, summary value. If the “center” of points on a _one dimensional_ number line is a single point, what is the “center” of points in a _two-dimensional_ cloud?
+[.lesson-point]
+Just as the mean, median, and mode can summarize data from a _single quantitative variable_, the “center” of points in a _two-dimensional_ cloud can be described by a @vocab{line of best fit}. Given a value for the @vocab{explanatory variable}, this line allows us to predict what the corresponding value for the @vocab{response variable} might be.
 
-What we need to do is find a _line_ -- called a @vocab{line of best fit}, or a @vocab{regression line} -- that is at the center of this cloud. Each point in our scatter plot “pulls” on the line, with points above the line yanking it up and points below the line dragging it down. Points that are really far away -- especially influential observations that are far out in the x direction —- pull on the line with more force. This line can be graphed on top of the scatter plot as a function, called the @vocab{predictor function}.
+[.lesson-instruction]
+--
```

Comment:
`.lesson-instruction` is used when there's something kids need to do. I don't see any instructions here. Does it need to be formatted as `.lesson-instruction`?

Reply:
@schanzer 
We are asking students to imagine something. I think that counts as instruction.